### PR TITLE
Getting Started Guide 🎉 

### PR DESCRIPTION
**Pull request by cprendergast19**
Created a Getting Started Guide

**Please check if the PR fulfills these requirements**
- [x] Followed the [Contributing](https://github.com/jaredwray/keyv/blob/main/CONTRIBUTING.md) guidelines.
- [x] Tests for the changes have been added (for bug fixes/features) with 100% code coverage.
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
for issue / feature #401

### CODE REVIEW

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -1 +1,129 @@
...
+
+```sh
+npm install --save @keyv/redis
+npm install --save @keyv/mongo
+npm install --save @keyv/sqlite
+npm install --save @keyv/postgres
+npm install --save @keyv/mysql
+npm install --save @keyv/etcd
+```
+
```
Comment (jaredwray) [inline docs/getting-started/index.md:29]: Would be good right here to provide the link to our other 3rd party storage adapters

File: docs/getting-started/index.md [hunk 2]
```diff
@@ -1 +1,129 @@
...
+## 3. Create a New Keyv Instance
+Pass your connection string if applicable. Keyv will automatically load the correct storage adapter.
+
```
Comment (jaredwray) [inline docs/getting-started/index.md:50]: An example would be great to show here

Comment (jaredwray) [inline docs/getting-started/index.md:50]: An example would be great to show here

File: docs/getting-started/index.md [hunk 3]
```diff
@@ -1 +1,129 @@
...
+deserialize | Function | N | A custom deserialization function. Default: JSONB.parse
+store | Storage adapter instance | N | The storage adapter instance to be used by Keyv. Default: new Map()
+adapter | String | N | Specify an adapter to use. e.g 'redis' or 'mongodb'. Default: undefined
+
+The following example shows how to create an Instance of Keyv either with or without a connection URI.
+
+```js
+const Keyv = require('keyv');
+
+// One of the following
```
Comment (jaredwray) [inline docs/getting-started/index.md]: I would just pick one of these storage adapters as we want them to walk through this and then they can fiddle around with all the options.

**Comment by jaredwray** [comment]
@cprendergast19 - this is really great progress so far. I think the next step after namespaces is to now take this code and add in compression using compress-gzip as an example. https://github.com/jaredwray/keyv/tree/main/packages/compress-gzip

After that it would be really cool to show that with this code example we an do `tiered` where we now do in memory and use the storage adapter as the backend.

**Comment by cprendergast19** [comment]
@jaredwray I've made all the updates based on the feedback so far.

**Pull request merged**